tests/Silverback.Integration.Tests.E2E/Kafka/CallbacksTests.cs

```diff
@@ -151,6 +156,177 @@ public async Task EndpointsConfiguredCallback_MessagePublished()
             kafkaTestingHelper.Spy.OutboundEnvelopes.Should().HaveCount(1);
         }
 
+        [Theory]
+        [InlineData(true, 6)] // There are 5 partitions and one message will be published, so in fact 6 times the end of the partition will be reached
+        [InlineData(false, 0)]
+        public async Task EndpointsConfiguredKafkaEndOfTopicPartitionReachedCallback(bool enablePartitionEof, int expectedEndOfPartitionReachedCount)
```

Comment:
* The name of the test doesn't fallow the standard UnitOfWork_Scenario_ExpectedResult convention.
* The test should also be split because in my opinion it tests two different scenarios/